crates/rolldown_binding/src/types/binding_normalized_input_options.rs

```diff
@@ -16,4 +19,30 @@ impl BindingNormalizedInputOptions {
   pub fn shim_missing_exports(&self) -> bool {
     self.inner.shim_missing_exports
   }
+
+  // Notice: rust's HashMap doesn't guarantee the order of keys, so not sure if it's a good idea to expose it to JS directly.
+  #[napi(getter)]
```

Comment:
Maybe we could return `Vec<(String, String)>` and convert to `{}` in js side? IIRC, Js map could preserve key order, also I assume this method is not frequently called in the real world.

Reply:
Yes. Let me do it in another PR. I just didn't want to write the normalization code form that time.